src/components/o-testimonials/index.scss

```diff
@@ -167,6 +167,14 @@ $o-testimonials-author-position-bottom: -40px;
   animation: fromRight 0.8s;
 }
 
+.o-testimonials--show-all-inline .o-testimonials__item-author {
```

Comment:
We are currently discussing CSS specificity and the goal of zero CSS specificity inclination #383

Reply:
@LucaMele @AndyOGo 
We need alternative option to display the slider in author. This is why I did this.
I can implement it in AEM without using the showAllInline tag if the way it is implemented doesn't comply with the pattern library. (zero CSS specificity inclination).

Reply:
is fine as it is @nakovskijosif